### ISSUE TITLE
Admin title overriding display title when creating new reusable block

### ISSUE THREAD

**Issue by nicolewaldrip** [CuBoulder/tiamat-theme#1155]
As part of the migration control cohort, Courtney MacDonald on the Libraries team is editing some of their content and experiencing an issue where the admin title is overriding the display title after she adds the block, despite entering the titles in the correct fields. This is happening on this site: https://live-ucbprod-libraries.pantheonsite.io/

![image](https://github.com/user-attachments/assets/9d76abab-b0dc-4d11-acee-2a6437749474)

Her specific report:
_I created a new reusable block to replace one that didn't migrate at all. i gave it a title to display, 'Hours Location and Contact Information' & i added the admin title 'map-contact-footer-text' in the section below the content. however when i add the block, the admin title shows in the Title spot every time. I've just been fixing it but this seems like something worth reporting._

**Comment by jnicholCU** [comment]
With the way reusable blocks work this is currently functioning as intended.

The Title is an option that can be changed per instance of the reusable block. It does not get saved to the Block Library.

The Admin Title is the machine name as well as the Block Description which does get saved into the Block Library.
That Block Description is the "default title" that gets used when a reusable block is placed.

In short, users can change the title of a block whenever it's placed and it won't affect the other uses of the block. 
In some ways that can be less convenient because you need to type out the Title each time you use the block, but it also means you can use that block content in different ways by applying different titles on different pages.

**Comment by jnicholCU** [comment]
Closing this out for now as backlog.
